components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,116 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumbs extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				--d2l-breadcrumbs-background-color: rgb(251, 252, 252);
+				display: block;
```

Comment:
If you set a `display` value for components, also provide CSS for the `hidden` attribute. [More information](https://github.com/BrightspaceUI/guide/wiki/LitElement-Best-Practices-&-Gotchas#-do-style-the-display-value-of-the-host)

Reply:
Done ✔️